epochStart/mock/builtInCostHandlerStub.go

```diff
@@ -13,7 +13,12 @@ func (b BuiltInCostHandlerStub) ComputeBuiltInCost(_ process.TransactionWithFeeH
 	return 1
 }
 
-// IsSpecialBuiltIn -
-func (b BuiltInCostHandlerStub) IsSpecialBuiltIn(_ process.TransactionWithFeeHandler) bool {
+// IsBuiltInFuncCall -
+func (b BuiltInCostHandlerStub) IsBuiltInFuncCall(_ process.TransactionWithFeeHandler) bool {
```

Comment:
can you change to pointer receiver everywhere?

Reply:
done.